index.html

```diff
@@ -0,0 +1,199 @@
+            class="second-nav-img"
+            alt="africa logo"
+            width="50"
+            height="50"
+        /></a>
+        <ul class="desktop-nav">
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li>
+            <a class="campaign-p" href="#">PC Africa</a>
+          </li>
+        </ul>
+      </div>
+    </header>
+    <div class="side-menu">
+      <div class="nav-menu">
+        <a href="index.html" class="side-menulogo"
+          ><img
+            src="images/africa.png"
+            alt="africa logo"
+            width="80"
+            height="80"
+            class="menu-logo"
+        /></a>
+        <img
+          src="images/close.png"
+          class="close-icon"
+          alt="closing icon"
+          width="20"
+          height="20"
+        />
+      </div>
+      <ul class="menu-ul">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#">PC Africa</a></li>
+      </ul>
+    </div>
+    <section class="first-mobile-container">
+      <div class="mobile-nav">
+        <img src="images/Union.png" alt="hambuger menu" />
+      </div>
+      <div class="first-mobile-content">
+        <p class="hello-africa">"Hello Africa"</p>
+        <h1>New Year Unity For Africa Concert 2023</h1>
+        <p class="mobile-information">
+          The African Concert Series fosters education across cultures, it is a
+          place where everyone, adults and children, can learn about the African
+          culture through its music.
+        </p>
+        <h2>31.12.2022(SAT)~1(SUN)</h2>
+        <p class="location">@ Carnivore Concert Center, Nairobi, Langata</p>
+      </div>
+    </section>
+    <section class="main-program">
+      <h2>Main Program</h2>
+      <div class="program">
+        <img src="images/wine.png" alt="" width="40" height="40" />
+        <h3>Opening</h3>
+
+        <p>
+          A unique musical show inspired this year by Africa, hosted by André
+          Manoukian.
+        </p>
+      </div>
+      <div class="program">
+        <img src="images/worldwide.png" alt="" width="40" height="40" />
+        <h3>Culture</h3>
+        <p>
+          Occasional photographer, lover of old photographs, frenetic
+          instagrammer.
+        </p>
+      </div>
+      <div class="program">
+        <img src="images/award.png" alt="" width="40" height="40" />
+        <h3>Awards</h3>
+
+        <p>
+          The winner of the Orange Foundation’s Orange Book Prize in Africa in
+          2019!
+        </p>
+      </div>
+      <div class="program">
+        <img src="images/music-note.png" alt="" width="40" height="40" />

```

Comment:
- [ ] I would kindly suggest you add an alternative text to all your images as this will help the User Experience to know what the image entails if eventually, the image link stops working 👍 👍 

> N.B: The page is accessible, images contain `alt` attributes and there are no images or elements blocking the text in both screen sizes.

  - Kindly add `alt text` to all the images where necessary.